src/word/hlxMediaHandler.js

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import mime from 'mime';
+import { createHash } from 'sha1-uint8array';
+
+import sizeOf from 'image-size';
+
+sizeOf.disableFS(true);
+
+// cache external urls
+const blobCache = {};
+
+/* Partial copy/reimplementation of https://github.com/adobe/helix-mediahandler/blob/main/src/MediaHandler.js
```

Comment:
@trieloff @tripodsan Want to confirm that neither of you have implemented this already in a cloudflare-friendly way.